assignment/week_3/RecommendedMenu/src/components/StartRecommend.jsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from "react";
+import styled from "styled-components";
+import OnBoarding from "./OnBoarding";
+import RecommendMenu from "./RecommendMenu";
+import ShowSelectedCategory from "./ShowSelectedCategory";
+
+const StartRecommend = (props) => {
+  const [isResetClicked, setIsResetClicked] = useState(false);
+  const [isStartClicked, setIsStartClicked] = useState(false);
+
+  const clickResetHandler = () => {
+    setIsResetClicked(true);
+    props.setSelectedCategory(false);
+    props.setStep(0);
```

Comment:
```suggestion
   const {setSelectedCategory, setStep} = props;
   
    setSelectedCategory(false);
    setStep(0);
```

이렇게 props 에서 구조분해할당 해서 쓰면 props.에 접근하지 않아도 되니까 가독성이 더 좋아지겠죠! - !

Reply:
네 !!!!!! `구조분해할당`. .... 제가 항상 어려워하고 헷갈려 했던 건데 저번 기수에도 서현이는 항상 `구조분해할당`을 많이 고려하고, 
리뷰 남길때도 구조분해할당 할 수 있는 부분에 대해 꼭 리뷰로 남겨주는 것 같더라고용?! 
저도 이번 기수에는 구조분해할당을 잘 활용할 수 있도록 코드에 많이 녹여내보겠슴돵 !!!